### ISSUE TITLE
row-class function is triggered every time after closing the modal 

### ISSUE THREAD

**Issue by larryu** [ratiw/vuetable-2#198]
@ratiw I met a problem. 
I have a vue table on the main screen, then if click the action button,a modal will be shown.
Every time I close the modal, the row-class function is triggered. I am wondering that because the data haven't been changed the vuetable should not be re-rendered. Am I right? Or Am I missing something?

**Comment by ratiw** [comment]
You're right. This has nothing to do with the modal at all (eventhough I have no idea what kind of modal you've implemented). It sounds more like the reactivity issue. Did you reference anything inside the row-class function?

**Comment by larryu** [comment]
@ratiw Yes. I referenced the row data and return different classes. But even though I comment out all the source code in this function, it still was called. I need to check whether somewhere is changing the data in my source code. I will keep you posted.
